* crashes any binary (mplayer) which uses the -ffast-math
the crash happens at the fxsave asm line.

diff -Nur gcc-4.1.0.orig/gcc/config/i386/crtfastmath.c gcc-4.1.0/gcc/config/i386/crtfastmath.c
--- gcc-4.1.0.orig/gcc/config/i386/crtfastmath.c	Wed Aug 17 06:07:06 2005
+++ gcc-4.1.0/gcc/config/i386/crtfastmath.c	Sat Mar  4 14:36:05 2006
@@ -72,6 +72,7 @@
   
       mxcsr |= MXCSR_FTZ;
 
+#if 0
       if (edx & FXSAVE)
 	{
 	  /* Check if DAZ is available.  */
@@ -99,6 +100,7 @@
 	  if (fxsave.mxcsr_mask & MXCSR_DAZ)
 	    mxcsr |= MXCSR_DAZ;
 	}
+#endif
 
       __builtin_ia32_ldmxcsr (mxcsr);
     }
